Library/Services/Implementation/NfieldInterviewersService.cs

```diff
@@ -169,6 +171,30 @@ public Task RemoveInterviewerFromFieldworkOfficesAsync(string interviewerId, str
             return Client.DeleteAsync(uri).FlattenExceptions();
         }
 
+        public async Task<string> LogsAsync(LogQueryModel query)
```

Comment:
I renamed it, but I have seen that `NfieldLocalUserService.LogSync` has the same name.
What do you think we will should do with the method for the users logs. If we rename it it will be a breaking change, because the `NfieldLocalUserService.LogSync' already has been released. 
Keep it as it is or change it because almost nobody will have time use it.

Reply:
For consistency sake we can add it and keep the old one, it can be just an extra interface member that points to the old method. We'll do it properly when we go to v2.